src/main/services/cloneFromBackupService.ts

```diff
@@ -190,8 +189,6 @@ const searchReplace = async (context: BackupMachineContext) => {
 	await siteDatabase.waitForDB(siteToSearchReplace);
 
 	await changeSiteDomain.changeSiteDomainToHost(siteToSearchReplace);
-
-	await siteProcessManager.restart(siteToSearchReplace);
```

Comment:
This one seems safe to remove. "Clone site from this backup" completes just fine without it.